Modules/xSharePoint/Modules/xSharePoint.Util/xSharePoint.Util.psm1

```diff
@@ -26,6 +26,10 @@ function Get-xSharePointAssemblyVersion() {
     )
     return (Get-Command $PathToAssembly).FileVersionInfo.FileMajorPart
 }
+function Get-UtilAsScriptBlock(){
```

Comment:
I don't see this function used anywhere and I'm not sure what it's purpose is, are you able to elaborate for me?